docs/troubleshooting/supabase-graph-sync-mode.md

```
@@ -0,0 +1,95 @@
+# Troubleshooting: Supabase + Graph 有効時の `GRAPH_SYNC_MODE` 起動エラー
+
+`STORAGE_BACKEND=supabase` かつ `GRAPH_ENABLED=true` の構成で、`GRAPH_SYNC_MODE` が `sync`（デフォルト値）のままになっていると、サーバー起動時（正確には `Settings` 構築時）に `ValidationError` が発生し、`memory_save` を含む全てのメモリ操作が失敗します。
+
+## 🚩 問題の概要
+
+`memory_save` などのツールを呼び出すと、以下のようなエラーが返ります。
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)